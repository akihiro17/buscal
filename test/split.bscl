var s: str = "this
is a
pen";

var array: array::<str> = [];
split(array, s);
for i in 0 to 2 {
  echo(format("{}: {}", i, array[i]));
}
