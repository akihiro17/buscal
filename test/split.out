0: this
1: is a
2: pen
